File: Trabalho de py e java 3.0/MaioreMenorN.py
N1=int(input("Digite sue número bb: "))
N2=int(input("Digite sue número bb: "))
N3=int(input("Digite sue número bb: "))

if N1 >= N2 and N1 >= N3:
    maior = N1
elif N2 >= N1 and N2 >= N3:
    maior = N2
else:
    maior = N3
if N1 <= N2 and N1 <= N3:
    menor = N1
elif N2 <= N1 and N2 <= N3:
    menor = N2
else:
    menor = N3
print(f"Maior: {maior}")
print(f"Menor: {menor}")

File: Trabalho de py e java 2.0/MaioreMenorN.java
import java.util.Scanner;
public class MaioreMenorN {
public static void main(String[]args){
Scanner scanner=new Scanner(System.in);
System.out.print("Digite seu número bb: ");
int N1=scanner.nextInt();
System.out.print("Digite seu número bb: ");
int N2=scanner.nextInt();
System.out.print("Digite seu número bb: ");
int N3=scanner.nextInt();
int Maior,Menor;

if(N1>=N2 && N1>=N3){Maior=N1;}
else if(N2 >= N1 && N2 >= N3){ Maior = N2;}
else{Maior=N3;}

if(N1 <= N2 && N1 <= N3){Menor = N1;}
else if (N2 <= N1 && N2 <= N3){Menor = N2;}
else{Menor = N3;}
System.out.println("Maior:"+Maior);
System.out.println("Menor:"+Menor);

    
scanner.close();
}
    
}
